### ISSUE TITLE
Empty file without header produces incomplete grid

### ISSUE THREAD

**Issue by j-tai** [sharkdp/bat#798]
If you disable the header with `--style` or with `BAT_STYLE`, an incomplete grid is shown when displaying an empty file:

```console
$ > empty.txt
$ bat --style=grid,numbers,snip empty.txt 
─────┬──────────────────────────────────────────
$
```

I have not set any `BAT_*` environment variables.

bat version: built from master (`bat --version` = 0.12.1)

**Comment by sharkdp** [comment]
Thank you for reporting this.

**Comment by sharkdp** [comment]
Closed by @eth-p in #841

**Comment by sharkdp** [comment]
This has been fixed in [bat 0.13](https://github.com/sharkdp/bat/releases/tag/v0.13.0).
